lib/octobox.rb

```diff
@@ -0,0 +1,31 @@
+require 'octobox/configurator'
+
+module Octobox
+  class << self
+    def config
+      @config ||= Configurator.new
+      if block_given?
+        yield @config
+      end
+      @config
+    end
+
```

Comment:
I don't have a good reason to want it.  It probably is better to have it separate